api/nakadi-event-bus-api.yaml

```diff
@@ -807,9 +807,13 @@ paths:
       security:
         - oauth2: ['nakadi.event_stream.read']
       description: |
-        Endpoint for committing offsets of the subscription. The client must commit at least once
-        every 60 seconds, otherwise Nakadi will consider the client to be gone and will close the
-        connection.
+        Endpoint for committing offsets of the subscription. If there is uncommited data, and no commits happen
+        for 60 seconds, then Nakadi will consider the client to be gone, and will close the connection. As long
+        as no events are sent, the client does not need to commit.
+
+        If the connection is closed, the client has 60 seconds to commit the events it received, from the moment
```

Comment:
I don't know if it's necessary, but this is true if the connection "is closed by the user" only. If the connection is closed by the server, it becomes immediately impossible to commit to that stream-id, as tested in https://techjira.zalando.net/browse/ARUHA-485